app/lib/features/cross_signing/pages/verification_request.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+
+class VerificationRequestPage extends StatelessWidget {
+  final String sender;
+  final bool passive;
+  final Function(BuildContext) onCancel;
+  final Function(BuildContext) onAccept;
+
+  const VerificationRequestPage({
+    super.key,
+    required this.sender,
+    required this.passive,
+    required this.onCancel,
+    required this.onAccept,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(15),
+      ),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Flexible(
+            flex: isDesktop ? 2 : 1,
```

Comment:
I am not convinced these `isDesktop` calls are that great, as in general we try to avoid that binary view of things: what if we are in a landscape mode on an iPad ... that is closer to the desktop but isn't covered by this flag .... I am okay with keeping this here for now, but in general this isn't a great pattern.